setup.py

```diff
@@ -354,8 +354,9 @@ def check_linker_need_libatomic():
 # By default, Python3 distutils enforces compatibility of
 # c plugins (.so files) with the OSX version Python was built with.
 # We need OSX 10.10, the oldest which supports C++ thread_local.
+# Python 3.9: Mac OS Big Sur sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET') returns int (11)
 if 'darwin' in sys.platform:
-    mac_target = sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET')
+    mac_target = str(sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET'))
     if mac_target and (pkg_resources.parse_version(mac_target) <
```

Comment:
I don't think this will work as intended - a `None` value for example will be converted to a string `'None'`. Maybe it's better to use:
```
pkg_resources.parse_version(str(mac_target))
```
which only gets executed after `if mac_target`.

Reply:
Agreed. @jorjun I think this PR will be good to go if you make this change.